main.go

```diff
@@ -35,24 +34,10 @@ func main() {
 	fmt.Println()
 	log.Infof("Running test")
 
-	var coverageExecutionFailed bool
-	if cfg.GenerateCodeCoverageFiles {
-		coverageExecutionFailed = coverage.executeCoverage(cfg.ProjectLocation, additionalParams)
-		coverage.exportCoverage(cfg.ProjectLocation)
+	outputBuffer, testErr := test.executeTest(cfg, additionalParams)
+	test.exportTestResults(cfg, outputBuffer)
```

Comment:
Exporting coverage is now inside `exportTestResults()`, because it's one of the multiple stuff that is exported after the test run.